sdml-core/src/model/definitions/classes.rs

```diff
@@ -169,6 +169,8 @@ impl TypeClassDef {
         extend_variables
             => variables, TypeVariable
     );
+
+    get_and_set!(pub span, set_span, unset_span => optional has_span, Span);
```

Comment:
This is actually performed by the `impl_has_source_span_for` macro, which implements the  `crate::model::HasSourceSpan` trait resulting in the methods `with_source_span`, `has_source_span`, `source_span`, `set_source_span` and `unset_source_span`.

Reply:
Have not noticed that macro. What lead me in the wrong direction is that module actually had such `get_and_set` available. Will change it to use the existing macro as it seems to be implemented in all/most places I need.